client/src/pages/Mates/MatesPage.tsx

```diff
@@ -1,12 +1,325 @@
+  };
+
+  const ModalForm = (
+    <Modal visible={isModalVisible} onOk={handleOk} onCancel={handleCancel}>
+      <Form layout="vertical">
+        <Form.Item label="Title">
+          <Input
+            value={editingSync.title}
+            placeholder="New Title"
+            onChange={(e) =>
+              changeFormValue(SyncModalEnum.TITLE, e.target.value)
+            }
+            required
+          />
+        </Form.Item>
+        <Form.Item label="Date">
+          <DatePicker
+            value={moment.utc(editingSync.ts)} // moment defaults to local time, which messes with JS's Date
+            onChange={(date, dateString) =>
+              changeFormValue(SyncModalEnum.DATE, dateString)
+            }
+          />
+        </Form.Item>
+        <Form.Item label="Details">
+          <Input.TextArea
+            autoSize
+            value={editingSync.details}
+            placeholder="Add details here..."
+            onChange={(e) =>
+              changeFormValue(SyncModalEnum.DETAILS, e.target.value)
+            }
+          />
+        </Form.Item>
+        <Form.Item label="Sync Status">
+          <Select
+            value={editingSync.syncStatus}
+            onChange={(value) =>
+              changeFormValue(SyncModalEnum.SYNC_STATUS, value)
+            }
+          >
+            {Object.entries(SyncStatusEnum).map(([syncStatus, value]) => (
+              <Select.Option key={syncStatus} value={value}>
+                {syncStatus}
+              </Select.Option>
+            ))}
+          </Select>
+        </Form.Item>
+      </Form>
+    </Modal>
+  );
+
+  return (
+    <>
+      {ModalForm}
+      <Row>
+        <Col span={18}>
+          <div
+            style={{
+              backgroundColor: "#F0F2F5",
+              height: "200px",
+              padding: "20px",
+            }}
+          >
+            <Title level={5}>M8</Title>
+            <Title>{mateName}</Title>
+            <Title level={5}>Syncs: Weekly</Title>
+          </div>
+          <div style={{ backgroundColor: "#F7F8FC", padding: "20px" }}>
+            <Row gutter={[24, 24]}>
+              {syncs
+                .sort((a, b) => b.ts.valueOf() - a.ts.valueOf())
+                .map((sync) => (
+                  <Col span={6} key={sync.id}>
+                    <SyncCard
+                      sync={sync}
+                      editSync={editSync}
+                      removeSync={removeSync}
+                    />
+                  </Col>
+                ))}
+            </Row>
+          </div>
+          <div style={{ position: "sticky", bottom: "0", right: "0" }}>
+            <Button
+              type="primary"
+              size="large"
+              icon={<PlusCircleOutlined />}
+              onClick={addSync}
+            >
+              Add Sync
+            </Button>
+          </div>
+        </Col>
+        <Col span={6} className="timeline-sidebar" style={{ height: "100vh" }}>
```

Comment:
Additionally, the timeline doesn't seem to go the full screen height:
![image](https://user-images.githubusercontent.com/9906905/137056657-cb92b276-616e-45a2-a907-7ac2534a0a6a.png)

try changing to
```
height: inherit
````

Reply:
dang, did not know this